common.tf

```diff
@@ -99,17 +100,24 @@ module "dr2_kms_key" {
   default_policy_variables = {
     user_roles = concat([
       module.ingest_parsed_court_document_event_handler_lambda.lambda_role_arn,
-      module.download_metadata_and_files_lambda.lambda_role_arn,
       module.ingest_mapper_lambda.lambda_role_arn,
       module.ingest_asset_opex_creator_lambda.lambda_role_arn,
       module.ingest_folder_opex_creator_lambda.lambda_role_arn,
       module.ingest_upsert_archive_folders_lambda.lambda_role_arn,
       module.ingest_parent_folder_opex_creator_lambda.lambda_role_arn,
       module.ingest_asset_reconciler_lambda.lambda_role_arn,
+
+      module.dr2_ingest_parsed_court_document_event_handler_lambda.lambda_role_arn,
+      module.dr2_ingest_mapper_lambda.lambda_role_arn,
+      module.dr2_ingest_asset_opex_creator_lambda.lambda_role_arn,
+      module.dr2_ingest_folder_opex_creator_lambda.lambda_role_arn,
+      module.dr2_ingest_upsert_archive_folders_lambda.lambda_role_arn,
+      module.dr2_ingest_parent_folder_opex_creator_lambda.lambda_role_arn,
+      module.dr2_ingest_asset_reconciler_lambda.lambda_role_arn,
+
       module.e2e_tests_ecs_task_role.role_arn,
```

Comment:
Is it ok that these don't start with `dr2`?

Reply:
They're ok. The only ones I've renamed are modules for queues and lambdas where we're running two copies in parallel. I don't really want to go through and change every module name. This can just be renamed as we're not bothered about it being destroyed and recreated.